src/views/CheckoutOverview.vue

```diff
@@ -75,7 +76,6 @@ export default class CheckoutOverview extends Vue {
             validityStartHeight,
             data: this.request.data,
             flags: this.request.flags,
-            networkId: this.request.networkId,
```

Comment:
unrelated

Reply:
It is related to updating the KeyguardClient dependency, to have the RPC features required. The API there changed and does not include the networkId anymore. Thus, to use the new KeyguardClient, this needs to be removed (will otherwise throw a typescript error).